### DISCUSSION TITLE
Welcome to pyarabic Discussions!

### DISCUSSION THREAD

**Discussion by linuxscout** [linuxscout/pyarabic#65] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by forzagreen** [comment]
Hi @linuxscout,
Thanks for the great work 👏 
I haven't worked yet with pyarabic, but I am interested in arabic NLP, and I maintain these 2 projects: [n2words](https://github.com/forzagreen/n2words) and [arabterm](https://github.com/forzagreen/arabterm)

**Comment by linuxscout** [comment]
Hi Wael (@forzagreen),
Thanks for your post,

- [PyArabic.number ](https://github.com/linuxscout/pyarabic/blob/master/pyarabic/number.py) contains similar function to [n2words](https://github.com/forzagreen/n2words).
- I have another project on Terms like [Nibras](https://nibras.sf.net).
You are welcome
